CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ Guide for installing them is [here](http://flake8.pycqa.org/en/latest/user/using
 #### Run mypy
 
 ``` bash
-mypy freqtrade
+mypy freqtrade tests
```

Comment:
This will not change the outcome ... `setup.cfg` defines the test folder as mypy exception.
```
[mypy-tests.*]
ignore_errors = True
```

removing that will cause 244 errors at this point in time.

Now while we could (and should) work on reducing this - we initially put it there after we discovered that would have had to add about 100 "#type: ignore" comments when we introduced mypy - mostly caused by mocks and similar things - so we decided to exclude tests for now from mypy.

I hope this (external stubs) did improve now so it's just a point of working on the errors - but i would keep contributing.md aligned to CI for the time being.

Reply:
In my understanding (as mypy docs says), ignore_errors only ignores non-fatal errors. fatal errors are still checked and reported. Again, if a contributor does not run it for tests, there can be failing CI runs after pushing commits.

Reply:
fatal errors are usually python syntax errors - which are then captured by running `pytest`.

While mypy does something similar (fails in case of syntax errors) - it will not tell you if you call a function with too many (or too few, or the wrong) arguments - as all these errors are ignored - so it has no benefit over running `mypy freqtrade` - and `pytest` (which is mandatory, too).

Reply:
okay